docs: improve Bitbucket cloud username (#29323)
renovatebot/renovate: lib/modules/platform/bitbucket/readme.md

Remember to:

> - Set the `username` for the bot account (which is _not_ your email address). You can find your username through "Personal Bitbucket settings" on the "Account settings" page for your account.

Reviewer: ```suggestion
- Set the `username` for the bot account (which is _not_ your email address). You can find your username by going to the "Personal Bitbucket settings" on your "Account settings" page
```

This may sound a bit silly, but we do not end _list items_ with a full stop. But _normal_ sentences should of course end with a full stop. :wink: 

I'm also rewriting the bit you added to be shorter.

Read the [Renovate style guide](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md) to learn more about the conventions for user-facing text.

Author: THanks for the pointer to the Renovate style guide, was not aware of it.

Reviewer: ```suggestion
- Set the `username` for the bot account (which is _not_ your email address). You can find your username by going to your Bitbucket "Account settings" page and then to "Personal Bitbucket settings"
```

How about this? It's a bit longer, but it lists the steps in more detail now.

Remember to:

> - Set the `username` for the bot account
> <!-- prettier-ignore -->
> !!! warning "Use the actual username not your emailaddress"
>     Bitbucker cloud requires your actual username and not your emailaddress. You can find your actual username by going to your settings using the cog. It should be listed as under "Username" in the section "Bitbucket profile settings"

Reviewer 2: ```suggestion
- Set the `username` for the bot account (which is not your email address). You can find your username under the "Bitbucket profile settings" page for your account.
```

Reviewer: Suggestion for the suggestion... :upside_down_face: 

We should probably _emphasize_ the _not_ part in: `(which is not your email address)`, to hammer the point home.

Author: I agree with suggested changes but would like to extend it a little to make the location more easily found since it is not at the first level, you need to click twice: first clik on the cog, then on "Personal Bitbucket Settings" which will land you on the  "Account settings" where you will be able to find the username.

Author: Updated the PR with my suggestion.
